src/ppspp_options.erl

```diff
@@ -228,8 +239,8 @@ get_chunk_addressing_method(Options) ->
 %% @end
 -spec get_content_integrity_check_method(options()) -> any().
 
-get_content_integrity_check_method(Options) ->
-    get(content_integrity_check_method, Options).
+get_content_integrity_check_method(#{content_integrity_check_method := Method} = _Options) ->
```

Comment:
Line 242 is too long: get_content_integrity_check_method(#{content_integrity_check_method := Method} = _Options) ->.